### PR TITLE
init nextjs

### PR DESCRIPTION
**Pull request by bluetch**
add next.js framework into angelhuang website

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function Home() {
         />
         <ol className="list-inside list-decimal text-sm text-center sm:text-left font-[family-name:var(--font-geist-mono)]">
           <li className="mb-2">
-            Get started by editing{" "}
+            Get started by editing{" "} AngelHuang
```
Comment (angelhuang0405) [inline app/page.tsx:17]: why name here

**Review by angelhuang0405 — CHANGES_REQUESTED** [review]
test

File: app/page.tsx (hunk 1 above)
Comment (bluetch) [reply app/page.tsx:17]: just a test, you can update by yourself

Comment (angelhuang0405) [reply app/page.tsx:17]: ok, thx

**Review by angelhuang0405 — APPROVED** [review]

**Pull request merged**